Files SnortSnarf-021111.1.orig/include/SnortSnarf/.HTMLAnomMemStorage.pm.swp and SnortSnarf-021111.1/include/SnortSnarf/.HTMLAnomMemStorage.pm.swp differ
Files SnortSnarf-021111.1.orig/include/SnortSnarf/.HTMLMemStorage.pm.swp and SnortSnarf-021111.1/include/SnortSnarf/.HTMLMemStorage.pm.swp differ
diff -urN SnortSnarf-021111.1.orig/include/SnortSnarf/HTMLAnomMemStorage.pm SnortSnarf-021111.1/include/SnortSnarf/HTMLAnomMemStorage.pm
--- SnortSnarf-021111.1.orig/include/SnortSnarf/HTMLAnomMemStorage.pm	2005-02-02 08:19:49.000000000 -0500
+++ SnortSnarf-021111.1/include/SnortSnarf/HTMLAnomMemStorage.pm	2005-02-02 08:50:44.740926336 -0500
@@ -263,7 +263,7 @@
         # already sorted
         my $end= $last-1;
         $end= $#{$arr} if ($end > $#{$arr});
-        return @arr->[($first-1)..$end];
+	return (\@arr)->[($first-1)..$end];
     } elsif (!defined($arr)) { # storage space had never been used
     	return ();
     }
diff -urN SnortSnarf-021111.1.orig/include/SnortSnarf/HTMLMemStorage.pm SnortSnarf-021111.1/include/SnortSnarf/HTMLMemStorage.pm
--- SnortSnarf-021111.1.orig/include/SnortSnarf/HTMLMemStorage.pm	2005-02-02 08:19:49.000000000 -0500
+++ SnortSnarf-021111.1/include/SnortSnarf/HTMLMemStorage.pm	2005-02-02 08:50:39.479797785 -0500
@@ -287,7 +287,7 @@
         # already sorted
         my $end= $last-1;
         $end= $#{$arr} if ($end > $#{$arr});
-        return @arr->[($first-1)..$end];
+	return (\@arr)->[($first-1)..$end];
     } elsif (!defined($arr)) { # storage space had never been used
     	return ();
     }
